src/icalendar/parser.py

```diff
@@ -171,6 +174,29 @@ def q_join(lst, sep=",", always_quote=False):
     return sep.join(dquote(itm, always_quote=always_quote) for itm in lst)
 
 
+def single_string_parameter(func):
```

Comment:
This did not show up in the docs PR preview. Not sure why.

Reply:
https://icalendar--856.org.readthedocs.build/en/856/search.html?q=single_string_parameter

Reply:
This is ok. It is used internally only and might be moved. I would not advertise it to people and so it is missing in `__all__`.